tests/__snapshots__/i18n.test.js.snap

```diff
@@ -0,0 +1,88 @@
+/en/getStaticProps/withRevalidate/2  /.netlify/functions/next_getStaticProps_withRevalidate_id  200
+/en/static/:id  /en/static/[id].html  200
+/es/getServerSideProps/all  /.netlify/functions/next_getServerSideProps_all_slug  200
+/es/getServerSideProps/all/*  /.netlify/functions/next_getServerSideProps_all_slug  200
+/es/getServerSideProps/static  /.netlify/functions/next_getServerSideProps_static  200
+/es/getServerSideProps/:id  /.netlify/functions/next_getServerSideProps_id  200
+/es/getStaticProps/static  /.netlify/functions/next_getStaticProps_static  200!  Cookie=__prerender_bypass,__next_preview_data
+/es/getStaticProps/with-revalidate  /.netlify/functions/next_getStaticProps_withrevalidate  200!  Cookie=__prerender_bypass,__next_preview_data
+/es/static/:id  /es/static/[id].html  200
+/getServerSideProps/all  /.netlify/functions/next_getServerSideProps_all_slug  200
+/getServerSideProps/all/*  /.netlify/functions/next_getServerSideProps_all_slug  200
+/getServerSideProps/static  /.netlify/functions/next_getServerSideProps_static  200
+/getServerSideProps/:id  /.netlify/functions/next_getServerSideProps_id  200
+/getStaticProps/1  /.netlify/functions/next_getStaticProps_id  200!  Cookie=__prerender_bypass,__next_preview_data
+/getStaticProps/1  /en/getStaticProps/1  200
+/getStaticProps/2  /.netlify/functions/next_getStaticProps_id  200!  Cookie=__prerender_bypass,__next_preview_data
+/getStaticProps/2  /en/getStaticProps/2  200
+/getStaticProps/static  /.netlify/functions/next_getStaticProps_static  200!  Cookie=__prerender_bypass,__next_preview_data
+/getStaticProps/static  /en/getStaticProps/static  200
+/getStaticProps/with-revalidate  /.netlify/functions/next_getStaticProps_withrevalidate  200!  Cookie=__prerender_bypass,__next_preview_data
+/getStaticProps/with-revalidate  /en/getStaticProps/with-revalidate  200
+/getStaticProps/withFallback/3  /.netlify/functions/next_getStaticProps_withFallback_id  200!  Cookie=__prerender_bypass,__next_preview_data
+/getStaticProps/withFallback/3  /en/getStaticProps/withFallback/3  200
+/getStaticProps/withFallback/4  /.netlify/functions/next_getStaticProps_withFallback_id  200!  Cookie=__prerender_bypass,__next_preview_data
+/getStaticProps/withFallback/4  /en/getStaticProps/withFallback/4  200
+/getStaticProps/withFallback/my/path/1  /.netlify/functions/next_getStaticProps_withFallback_slug  200!  Cookie=__prerender_bypass,__next_preview_data
+/getStaticProps/withFallback/my/path/1  /en/getStaticProps/withFallback/my/path/1  200
+/getStaticProps/withFallback/my/path/2  /.netlify/functions/next_getStaticProps_withFallback_slug  200!  Cookie=__prerender_bypass,__next_preview_data
+/getStaticProps/withFallback/my/path/2  /en/getStaticProps/withFallback/my/path/2  200
+/getStaticProps/withFallback/:id  /.netlify/functions/next_getStaticProps_withFallback_id  200
+/getStaticProps/withFallback/:slug/*  /.netlify/functions/next_getStaticProps_withFallback_slug  200
```

Comment:
i18n for pages with fallback is missing. There are no entries here for `en` and `es` versions of this redirect. ⚠